src/config.mjs

```diff
@@ -34,7 +34,7 @@ const config = {
     password: optional('FRAPI_POSTGRES_PASSWORD', [], undefined),
   },
   anope: {
-    database: required('FRAPI_ANOPE_DATABASE', [], 'anope'),
+    database: optional('FRAPI_ANOPE_DATABASE', [], undefined),
```

Comment:
you know funny enough, I did this EXACT same thing when I first started working on API stuff, but changed my mind because I wanted things to be closer to production with working nicks and such.